### DISCUSSION TITLE
Why do LOG_LEVEL and DEBUG_FLAGS require user consent?

### DISCUSSION THREAD

**Discussion by jdevera** [criteo/command-launcher#145] [Q&A]
I'm doing an in-depth review of the docs and found this puzzling. What's sensitive about the log level and debug flags that would require user consent before passing to commands? Seems pretty standard procedure to me. What am I missing here?

**Comment by bhou-crto** [comment]
Hi @jdevera,
Firstly, Happy new year!

You are right; these two "resources" no longer need user consent. In the past, the implementation was tightly coupled with Criteo's internal business, which requires user consent on these two resources. With the current open source context, it doesn't make sense to keep them this way.

I will push a review to clean up some of the obsolete code and update the doc.
